zulipterminal/model.py

```diff
@@ -458,10 +460,13 @@ def update_stream_message(self, topic: str, message_id: int,
 
     def fetch_custom_emojis(self) -> NamedEmojiData:
         response = self.client.get_realm_emoji()
-        custom_emojis = {emoji['name']: {'code': emoji_code,
-                                         'type': 'realm_emoji'}
-                         for emoji_code, emoji in response['emoji'].items()
-                         if not emoji['deactivated']}  # type: NamedEmojiData
+        custom_emojis: NamedEmojiData = {
+            emoji['name']: {
+                'code': emoji_code,
+                'type': 'realm_emoji'
+                }
+            for emoji_code, emoji in response['emoji'].items()
+            if not emoji['deactivated']}
```

Comment:
Did com2ann generate this new layout? We're moving towards a style where large bracketed code have the closing one on the starting line, though the original version isn't ideal in this respect either.

Reply:
Meaning to be like that, right?

```
        custom_emojis: NamedEmojiData = {
            emoji['name']: {
                'code': emoji_code,
                'type': 'realm_emoji'
            }
            for emoji_code, emoji in response['emoji'].items()
            if not emoji['deactivated']
        }
```

Reply:
Right :+1: Also trailing commas, but that's simpler to quickly fix :)

Reply:
@neiljp Done :)